### ISSUE TITLE
ReferenceError: Cannot access 'c' before initialization

### ISSUE THREAD

**Issue by christianbundy** [sure-thing/presta#19]
I cloned this repository and tried to run it. I'm sure I'm using it wrong, but this is a strange error. I'm on 4bd16a2bafd50c97d94c47a65de51b53974d22bc.

```console
$ node -r esm --trace-warnings presta.js
(node:79192) UnhandledPromiseRejectionWarning: ReferenceError: Cannot access 'c' before initialization
    at Proxy.create (/Users/christianbundy/Source/presta/lib/config.js:33:14)
    at /Users/christianbundy/Source/presta/cli.js:24:31
    at Sade.parse (/Users/christianbundy/Source/presta/node_modules/sade/lib/index.js:190:56)
    at Object.<anonymous> (/Users/christianbundy/Source/presta/cli.js:53:6)
    at Generator.next (<anonymous>)
    at createWarningObject (internal/process/warning.js:161:9)
    at process.emitWarning (internal/process/warning.js:131:15)
    at process.<anonymous> (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:289642)
    at process.<anonymous> (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:289518)
    at process.<anonymous> (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:284879)
    at Object.apply (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:199341)
    at emitUnhandledRejectionWarning (internal/process/promises.js:168:15)
    at processPromiseRejections (internal/process/promises.js:247:11)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
    at process.runNextTicks [as _tickCallback] (internal/process/task_queues.js:62:3)
(node:79192) ReferenceError: Cannot access 'c' before initialization
    at Proxy.create (/Users/christianbundy/Source/presta/lib/config.js:33:14)
    at /Users/christianbundy/Source/presta/cli.js:24:31
    at Sade.parse (/Users/christianbundy/Source/presta/node_modules/sade/lib/index.js:190:56)
    at Object.<anonymous> (/Users/christianbundy/Source/presta/cli.js:53:6)
    at Generator.next (<anonymous>)
    at bl (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:245412)
    at kl (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:247659)
    at u (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:287740)
    at o (/Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:287137)
    at /Users/christianbundy/Source/presta/node_modules/esm/esm.js:1:284879

```

```js
{
  npm: '6.14.8',
  ares: '1.16.1',
  brotli: '1.0.9',
  cldr: '37.0',
  icu: '67.1',
  llhttp: '2.1.3',
  modules: '83',
  napi: '7',
  nghttp2: '1.41.0',
  node: '14.15.1',
  openssl: '1.1.1g',
  tz: '2019c',
  unicode: '13.0',
  uv: '1.40.0',
  v8: '8.4.371.19-node.17',
  zlib: '1.2.11'
}
```

**Comment by christianbundy** [comment]
Quick note: Hi! I was just kicking the tires on this repo, not using it for anything important, so please don't worry about this error. Just wanted to make an issue so that the error was written down. 🎬

**Comment by estrattonbailey** [comment]
Yo! Thanks for having a look! Yeah I broke this before I left for Thanksgiving and forgot to push it up 😅 should be fixed by f8f4a50685316ef69f633ace5b33167fb506c55a and available in 0.17.2
